src/electron/device-connect-view/device-connect-view-initializer.ts

```diff
@@ -1,28 +1,48 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { AppInsights } from 'applicationinsights-js';
 import { remote } from 'electron';
 import * as ReactDOM from 'react-dom';
 
 import { UserConfigurationActions } from '../../background/actions/user-configuration-actions';
 import { getPersistedData, PersistedData } from '../../background/get-persisted-data';
 import { IndexedDBDataKeys } from '../../background/IndexedDBDataKeys';
+import { InstallationData } from '../../background/installation-data';
 import { UserConfigurationStore } from '../../background/stores/global/user-configuration-store';
+import { getTelemetryClient } from '../../background/telemetry/telemetry-client-provider';
+import { TelemetryEventHandler } from '../../background/telemetry/telemetry-event-handler';
+import { TelemetryLogger } from '../../background/telemetry/telemetry-logger';
+import { TelemetryStateListener } from '../../background/telemetry/telemetry-state-listener';
 import { initializeFabricIcons } from '../../common/fabric-icons';
 import { getIndexedDBStore } from '../../common/indexedDB/get-indexeddb-store';
 import { IndexedDBAPI, IndexedDBUtil } from '../../common/indexedDB/indexedDB';
+import { ElectronAppDataAdapter } from '../adapters/electron-app-data-adapter';
+import { ElectronStorageAdapter } from '../adapters/electron-storage-adapter';
 import { DeviceConnectViewRenderer } from './device-connect-view-renderer';
 
 initializeFabricIcons();
 
 const indexedDBInstance: IndexedDBAPI = new IndexedDBUtil(getIndexedDBStore());
 const userConfigActions = new UserConfigurationActions();
+const storageAdapter = new ElectronStorageAdapter(indexedDBInstance);
+const appDataAdapter = new ElectronAppDataAdapter();
 
-const indexedDBDataKeysToFetch = [IndexedDBDataKeys.userConfiguration];
+const indexedDBDataKeysToFetch = [IndexedDBDataKeys.userConfiguration, IndexedDBDataKeys.installation];
 
 // tslint:disable-next-line:no-floating-promises - top-level entry points are intentionally floating promises
 getPersistedData(indexedDBInstance, indexedDBDataKeysToFetch).then((persistedData: Partial<PersistedData>) => {
-    // tslint:disable-next-line:no-unused-variable
+    const installationData: InstallationData = persistedData.installationData;
+
+    // wireup telemetry
```

Comment:
is this comment necessary?

Reply:
Will remove comment as it doesn't provide any important info